tests/ui/string_to_string.rs

```diff
@@ -0,0 +1,8 @@
+#![warn(clippy::string_to_string)]
+
+fn main() {
+    let mut message = String::from("Hello");
+    let mut v = message.to_string();
+    v = String::from("Hello"); // to avoid redundant clone error
+    message.push_str("World!"); // to avoid redundant clone error
```

Comment:
Just add an `allow` for the `redundant_clone`lint to this test file.

Reply:
ok